### DISCUSSION TITLE
Performance load - strange behaviour - missing items

### DISCUSSION THREAD

**Discussion by plamen-dzhorev** [probonopd/MiniDexed#537] [Q&A] [answered]
Hello,

I've noticed some discrepancies in performance loading. I am using the collection from the repo which contains 130 items. When I cycle through all items, some are missing at random positions (at least half of the items are not loaded). Checked the screen messages and there were no visible errors. All the ini files seem ok, so I decided to try with different folder content. I deleted all of the performances which were successfully loaded and put some of the missing performances. First time I tried with 5 files and all loaded - no files missing. Then, I increased the count to 10 and again everything was ok. After that, increased the files to 50 and the issue happened - 10 were missing. I cannot find any pattern in the missing ones, looks like they are randomly skipped for all numbers. 
The issue happens with different builds - tested with last released and a build from 25th August.
Everything else works as expected - voices, TGs, etc, at least the functions I've used.
My hardware is:
Rpi3A+
LCD 1602 without i2c
PCM5102 DAC
Rotary encoder KY040 with 10 nF caps

Anybody has an idea what could be wrong?

**Comment by Banana71** [comment, marked as answer]
Hello Plamen,

I haven't had this problem before. Please try the following version: https://github.com/probonopd/MiniDexed/actions/runs/5860156515
MiniDexed_2023-08-14-48d7112

But um, please note one of my quirks.
I sorted the performances into categories and left spares (gaps) between the categories so that I can better sort new sounds into the categories.
You probably have a version with a total of 114 Performances on the SD card and the last Performance is the "130-Multitimbral".
Until recently, nobody noticed it, but after one of the last updates, the numbers of the performances are also displayed and there are gaps. I try to fill the gaps with new sounds as soon as possible.

Unfortunately, I did not delete the performance 000055_Choir 1TG_Dry.ini and it can disappear from the SD card.

**Reply by plamikcho** [reply]
Hello Peter,

Thanks for the fast response! The version from the link solved the issue and now all performances are loaded. On the broken versions I also noticed that there is an empty item (without name) near the end (a few items before 130-Multitimbral). Thanks again for the help and for the excellent sound you have created!

Plamen (this is my personal account)

**Comment by Banana71** [comment]
Great that everything works now.  
Can you please let me know which version you were having the problem with.  (link or date)

**Reply by plamikcho** [reply]
Sorry for the noise - the broken version is an old one - MiniDexed_2022-08-25-dcbd04c. This version is the last release with a date except the continuous build. I tested again with MiniDexed_2023-08-10-375900b and MiniDexed_2023-08-14-48d7112 and both are ok, all 114 performances are loaded.

**Reply by probonopd** [reply]
Please use the latest continous build, it should contain the fix. Thanks!

**Reply by plamikcho** [reply]
Yes, I will for sure. I've created a mess while trying different versions but I was pretty sure that the problem is at my side - because nobody has reported it here. Thanks again for this great project and for your support!
